### PR TITLE
x11-misc/clipmenu: initial commit

### PR DESCRIPTION
**Pull request by ferki**
This PR adds x11-misc/clipmenu-6.0.1 to the portage tree, along with x11-misc/clipnotify-1.0.2 as dependency.

I'd also would like to step up as proxied maintainer for these packages.

Please review, then either merge or let me know if something needs to be modified first.

Closes: https://bugs.gentoo.org/722228

### CODE REVIEW

File: x11-misc/clipnotify/clipnotify-1.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	x11-libs/libX11
+	x11-libs/libXfixes
+"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
```
Comment (DarthGandalf) [inline x11-misc/clipnotify/clipnotify-1.0.2.ebuild:20]: DEPEND and BDEPEND packages are built for different architectures when cross-compiling

Comment (ferki) [reply x11-misc/clipnotify/clipnotify-1.0.2.ebuild:20]: @DarthGandalf: I'm not entirely sure what's the expected action in response to that remark from my side. What would you suggest using instead that would be acceptable?

Comment (DarthGandalf) [reply x11-misc/clipnotify/clipnotify-1.0.2.ebuild:20]: Please review https://devmanual.gentoo.org/general-concepts/dependencies/index.html and decide what dependency goes to where. Perhaps, by removing BDEPEND if the build system doesn't need anything installed.

**Comment by ferki** [comment]
I've made the following changes:

- rebased on top of current master
- removed BDEPEND from x11-misc/clipnotify
- introduced a local variable for the install loop (thanks @jonasstein!)
- bumped x11-misc/clipmenu to latest version (6.1.0), which was just released meanwhile

**Review by monsieurp — APPROVED** [review]

**Pull request closed without merging**